### PR TITLE
QGaussLobatto: add fallback on regular Gauss quadrature

### PR DESCRIPTION
**Pull request by jwpeterson**
Refs #2545

### CODE REVIEW

**Comment by jwpeterson** [comment]
@hchen139 if you are able to test this with your application that would be great, I did not test it yet.

**Comment by hchen139** [comment]
John, this is exactly what I have in my code. It worked for me.

File: src/quadrature/quadrature_gauss_lobatto_2D.C [hunk 1]
```diff
@@ -42,12 +43,21 @@ void QGaussLobatto::init_2D(const ElemType, unsigned int)
...
+      // We fall back on a Gauss type rule for other types of elements,
+      // but we warn the user (once) that we are doing this instead of
+      // silently switching out quadrature rules on them.
     default:
-      libmesh_error_msg("Element type not supported:" << Utility::enum_to_string(_type));
+      {
+        libmesh_warning("Warning: QGaussLobatto falling back on QGauss rule "
+                        "for unsupported Elem type: " << Utility::enum_to_string(_type));
+
+        QGauss gauss_rule(_dim, _order);
```
[the remark this answers is not in the document]
Comment (jwpeterson) [reply src/quadrature/quadrature_gauss_lobatto_2D.C:54]: I’m pretty sure the two things should be equivalent. Is that not the case? Other things being equal, I’d prefer to use the class member over a “magic” number whenever possible.

**Comment by jwpeterson** [comment]
@hchen139 there was a test failure in `samplers/base.errors/getLocalSamples_max` but I don't see how it can be related to this PR so I invalidated it... hopefully it passes on the 2nd try.

**Pull request merged**